DTAConfig/OptionPanels/AudioOptionsPanel.cs

```diff
@@ -10,6 +10,20 @@ namespace DTAConfig.OptionPanels
 {
     class AudioOptionsPanel : XNAOptionsPanel
     {
+        private const int VOLUME_MIN = 0;
+        private const int VOLUME_MAX = 10;
+        private const float VOLUME_SCALE_FLOAT = 10.0f;
+        private const int VOLUME_SCALE_INT = 10;
+        private const double VOLUME_SCALE_DOUBLE = 10;
```

Comment:
Could do

```suggestion
        private const int VOLUME_SCALE = 10;
```
and then just use `/ (float)VOLUME_SCALE_INT` and `/ (double)VOLUME_SCALE_INT` where needed.